admin/services/school-impersonation.service.js

```diff
@@ -12,24 +12,29 @@ const schoolImpersonationService = {}
  * @param {string} dfeNumber
  * @returns {Object}
  */
-schoolImpersonationService.validateImpersonationForm = async (user, dfeNumber) => {
+schoolImpersonationService.processImpersonationForm = async (user, dfeNumber) => {
```

Comment:
again, odd choice of method name.  maybe `setSchoolImpersonation()` ?

Reply:
@GuyHarwood updated